app/scripts/modules/core/help/helpField.component.ts

```diff
@@ -87,21 +88,37 @@ class HelpFieldComponent implements ng.IComponentOptions {
     content: '@',
     placement: '@',
     expand: '=',
+    label: '@'
   };
   public controller: any = HelpFieldCtrl;
   public template = `
-    <div ng-if="$ctrl.expand && $ctrl.contents.content"
-         class="help-contents small"
-         ng-bind-html="$ctrl.contents.content"></div>
-    <a href class="help-field" ng-if="!$ctrl.expand && $ctrl.contents.content"
-            uib-popover-template="$ctrl.popoverTemplate"
-            ng-mouseenter="$ctrl.showPopover()"
-            ng-mouseleave="$ctrl.hidePopover(true)"
-            popover-placement="{{$ctrl.contents.placement}}"
-            popover-is-open="$ctrl.displayPopover"
-            popover-trigger="none">
-      <span class="small glyphicon glyphicon-question-sign"></span>
-    </a>
+    <div style="display: inline-block;">
+      <div class="text-only" ng-if="$ctrl.label">
```

Comment:
So it actually applied to the outer `div` but it was really only necessary on the second `div` if the `label` property was NOT defined, so I moved it.